libs/ts-rest/fastify/src/lib/ts-rest-fastify.ts

```diff
@@ -176,6 +176,8 @@ const requestValidationErrorHandler = (
       } else {
         return handler(err, request, reply);
       }
+    } else {
+      throw err;
```

Comment:
I receiving here

```
http: LogLevel.ERROR: ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response')) while doing a GET request to URL: http://localhost:4000/
```

I prefer to get 500 with error message or even better be able to override it by my own handler.